src/translators/ObjectsTranslator.ts

```diff
@@ -185,42 +188,51 @@ export abstract class ObjectsTranslator {
             for (let i = 0; i < numTableModifications; i++) {
                 const objectDefinition = []; // object definition will store one or more modification objects
 
-                const originalId = outBufferToJSON.readChars(4),
-                    customId = outBufferToJSON.readChars(4),
-                    modificationCount = outBufferToJSON.readInt();
-
-                for (let j = 0; j < modificationCount; j++) {
-                    const modification: Modification = {
-                        id: '',
-                        type: ModificationType.string,
-                        level: 0,
-                        column: 0,
-                        value: {}
-                    };
-
-                    modification.id = outBufferToJSON.readChars(4);
-                    modification.type = this.varTypes[outBufferToJSON.readInt()]; // 'int' | 'real' | 'unreal' | 'string',
+                const originalId = outBufferToJSON.readChars(4);
+                const customId = outBufferToJSON.readChars(4);
+                let sets = 1;
```

Comment:
Also can you clarify since I haven't looked at a 1.33 object data diff: the entire modification table is now wrapped in an additional loop? What do the sets represent?

Reply:
I'm not really sure, I'll have to do some research,  this was copied from mdx-model-viewer

Reply:
Yea if you could also dig around to figure out what it is, that would be useful.

Here's an experiment I did:
1. Create a new map in World Editor
2. Modify the (human) Peasant original object - edit just one field, in my case base damage
3. Annotate the bytes in the `.w3u` file to check for new data

I am seeing some new bytes that aren't accounted for in the version 2 of the parser:
```
03 00 00 00  :: version 3

01 00 00 00  :: num modification tables

:: modification table 1 - Peasant
68 70 65 61  :: "hpea" (human peasant)
00 00 00 00  :: custom id - 0 = original object
01 00 00 00  :: modifications to this object: 1

  :: modification 1 - Peasant - Base Damage
  00 00 00 00  :: mod id
  01 00 00 00  :: mod type - 1 = real
  75 61 31 62  :: "ua1b" (unit attack 1 - base damage)
  00 00 00 00  :: <<<< NEW 4 BYTES ??
  0C 00 00 00  :: value = 12 (base damage updated to 12)
  68 70 65 61  :: "hpea"
  00 00 00 00  :: <<<< NEW 4 BYTES ??
```